cmd/snap/cmd_find.go

```diff
@@ -113,15 +113,24 @@ func (x *cmdFind) Execute(args []string) error {
 	}
 
 	// magic! `snap find` returns the featured snaps
-	if x.Positional.Query == "" && x.Section == "" {
+	showFeatured := (x.Positional.Query == "" && x.Section == "")
+	if showFeatured {
 		x.Section = "featured"
+		fmt.Fprintf(Stdout, i18n.G("No search term specified, showing the featured snaps:\n\n"))
```

Comment:
This is a run-on sentence. The simplest fix would be starting a new sentence instead of the comma.